crates/re_types_builder/src/codegen/docs/mod.rs

```diff
@@ -96,6 +94,12 @@ on [Entities and Components](../../concepts/entity-component.md).",
                 r"Data types are the lowest layer of the data model hierarchy. They are re-usable types used by the components.",
                 &datatypes,
             ),
+            (
+                ObjectKind::View,
+                4,
+                r"Views are responsible for creating visual representations of archetypes. This page lists all built-in views.",
```

Comment:
That sounds more like you're describing visualizers. We're actually populating the viewport of the viewer here and then show stuff in them, not even necessarily archetypes. How about:

```suggestion
                r"Views are the panels shown in the viewer's viewport and the primary means of inspecting & visualizing previously logged data. This page lists all built-in views.",
```